artifacthub/add-ons-http/0.0.1/artifacthub-pkg.yml

```diff
@@ -0,0 +1,26 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.0.1
+name: keda-add-ons-http
+displayName: KEDA HTTP
+createdAt: 2021-02-09T10:00:00Z
+description: Event-based autoscaler for HTTP workloads on Kubernetes
+license: Apache-2.0
+homeURL: https://github.com/kedacore/http-add-on
+keywords:
+  - keda
+  - http
+links:
+  - name: GitHub
+    url: https://github.com/kedacore/http-add-on
+readme: |
+  Alpha 1 is the first official release of the KEDA HTTP Add On, and is intended as a preview for developers. With this release, we're inviting community members to try out the technology, report bugs, and generally give us feedback.
+changes:
+  - Initial version
+maintainers:
+  - name: Sample maintainer
+    email: sample-maintainer@email.com
```

Comment:
This should be set probably, right?

Reply:
Removed it since it's not mandatory

Reply:
https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml